### PR TITLE
Power and Energy Hooks

### LINKED ISSUE #85: Hooks Regarding Energy Consumption
Add additional hooks regarding energy consumption such as:
- `RollPass`
  - power, energy per ton
- `PassSequence`
  - total power, total energy consumption

Remove `total` keyword from `PassSequence` hooks, to enable simplified nesting.

### PR DESCRIPTION
**Pull request by axtimhaus**
Closes #85 

Energy consumption requires density!

### CODE REVIEW

File: pyroll/core/unit/hookimpls.py [hunk 1]
```diff
@@ -64,3 +64,24 @@ def out_x(self: Unit.OutProfile):
...
 def out_t(self: Unit.OutProfile):
     return self.unit.in_profile.t + self.unit.duration
+
+
+@Unit.power(trylast=True)
+def default_power(self: Unit):
+    return 0
+
+
+@Unit.volume_throughput
```
Comment (ChRen95) [inline pyroll/core/unit/hookimpls.py:74]: Should be named volume_flux

File: pyroll/core/unit/unit.py [hunk 2]
```diff
@@ -33,6 +33,18 @@ class Unit(HookHost):
     surface_area = Hook[float]()
     """Surface area of workpiece within the unit."""
 
+    volume_throughput = Hook[float]()
```
Comment (ChRen95) [inline pyroll/core/unit/unit.py:36]: Isn't really a volume, so either include standard length of 1 or we should think about renaming

**Review by ChRen95 — CHANGES_REQUESTED** [review]
Should rename Hook Volume Throughput to Volume Flux since this is generally used in literature.

**Comment by axtimhaus** [comment]
Should name the mass and volume flux equally.
Throughput is also a valid term, but more from the economical point of view. But I agree with using flux.

**Review by ChRen95 — APPROVED** [review]

**Pull request merged**